pkg/kots/lint.go

```diff
@@ -364,12 +364,17 @@ func lintRenderContent(specFiles SpecFiles) ([]LintExpression, SpecFiles, error)
 		lintExpressions = append(lintExpressions, lintExpression)
 	}
 
+	builder, err := getTemplateBuilder(config)
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "failed to get  builder")
```

Comment:
This error message looks odd.

Reply:
thanks! that should be "failed to get template builder"